### PR TITLE
Use hashes in webpack JS assets

### PR DESCRIPTION
**Pull request by bogas04**
* [ ]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [ ]  Ran `npm test` & fixed newly introduced lint errors.
* [ ]  Checked console for errors.

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

File: server/template.js [hunk 1]
```diff
@@ -110,8 +136,14 @@ export default ({ url, bodyClass, title, description }) => marinate`
...
   <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Object.entries,Array.from,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Map,Set,requestAnimationFrame,Array.prototype.@@iterator"></script>
 
+<<<<<<< HEAD
+  <script src="${manifest['vendor.js']}"></script>
+  <script src="${manifest['app.js']}"></script>
+</body>
+=======
   <!-- Source Code -->
   ${scriptsHTML}
+>>>>>>> 169583a2dc05864018ec6e2212109621b9ae8b25
```
Comment (bogas04) [inline server/template.js]: This doesn't seem to be right. Bad merge?

File: server/template.js [hunk 2]
```diff
@@ -19,6 +28,23 @@ export default ({ url, bodyClass, title, description }) => marinate`
...
+  <link async rel="preload" href="/assets/fonts/GurbaniAkharHeavyTrue.ttf?v=1" as="font">
+  <link async rel="preload" href="/assets/fonts/AnmolLipiSG.ttf?v=1" as="font">
+
+  <!-- Preload JavaScript -->
+  <link async rel="preload" href="${manifest['Search.js']}" as="script">
+  <link async rel="preload" href="${manifest['Shabad.js']}" as="script">
+
+  <!-- Preload Images -->
+  <link async rel="preload" href="/assets/images/sttm_logo.png" as="image">
+  <link async rel="preload" href="/assets/images/logo-192x192.png" as="image">
```
Comment (bogas04) [inline server/template.js]: Bad merge? Dev has automated strings for this.

File: server/template.js [hunk 3]
```diff
@@ -110,8 +136,14 @@ export default ({ url, bodyClass, title, description }) => marinate`
   <!-- Polyfills -->
   <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Object.entries,Array.from,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Map,Set,requestAnimationFrame,Array.prototype.@@iterator"></script>
 
+<<<<<<< HEAD
+  <script src="${manifest['vendor.js']}"></script>
+  <script src="${manifest['app.js']}"></script>
+</body>
+=======
   <!-- Source Code -->
   ${scriptsHTML}
```
Comment (bogas04) [inline server/template.js]: `scriptsHTML` handles the script tags, you just need to update the array below.

File: server/template.js [hunk 4]
```diff
@@ -110,8 +136,14 @@ export default ({ url, bodyClass, title, description }) => marinate`
   <!-- Polyfills -->
   <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Object.entries,Array.from,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Map,Set,requestAnimationFrame,Array.prototype.@@iterator"></script>
 
+<<<<<<< HEAD
+  <script src="${manifest['vendor.js']}"></script>
+  <script src="${manifest['app.js']}"></script>
+</body>
+=======
```
Comment (bogas04) [inline server/template.js]: This isn't required as `scriptsHTML` takes care of this.

File: server/template.js (hunk 3 above)
Comment (Gauravjeetsingh) [reply server/template.js]: Couldn't get you here. Which array are we talking about?

File: server/template.js (hunk 4 above)
Comment (Gauravjeetsingh) [reply server/template.js]: I removed these lines, is there any other change required?

File: server/template.js (hunk 3 above)
Comment (bogas04) [reply server/template.js]: Check at the bottom of the file and look for `scripts` array. 

https://github.com/KhalisFoundation/sttm-web/blob/dev/server/template.js#L173

File: server/template.js (hunk 2 above)
Comment (bogas04) [reply server/template.js]: Remove all these lines as well. Check the *HTML variables, they're handling these now.

File: server/template.js (hunk 4 above)
Comment (bogas04) [reply server/template.js]: <<<<< HEAD is part of git conflict. We need to resolve that. And the resolution is to remove the script part as `scriptHTML` is gonna put them.

File: server/template.js (hunk 3 above)
Comment (Gauravjeetsingh) [reply server/template.js]: @bogas04 Can you check now, if all the required changes are done?

**Pull request merged**